infra/modules/providers/azure/storage-account/README.md

```diff
@@ -0,0 +1,118 @@
+# Azure Storage
+
+This Terraform based `storage-account` module grants templates the ability to configure and deploy cloud `storage containers` along with a `storage account` using Microsoft's _**Azure Storage**_ service.
+
+In addition, this module offers both "authentication" and "authorization" features:
+
+- For "authentication", this module automatically enrolls the deployed `storage account` into Microsoft's _**Managed Identities**_ service.
```

Comment:
[ nit ] adding quotes around `authentication` and `authorization` makes it sound dubious :D

Reply:
Quotes were removed in the latest commit.